src/test/java/kdt/springbootjpa/customer/CustomerPersistenceTest.java

```diff
@@ -0,0 +1,101 @@
+package kdt.springbootjpa.customer;
+
+import kdt.springbootjpa.customer.entity.Customer;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.EntityTransaction;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DataJpaTest
+public class CustomerPersistenceTest {
+
+    @Autowired
+    EntityManagerFactory entityManagerFactory;
```

Comment:
```suggestion

	@Autowired
    private EntityManagerFactory entityManagerFactory;
	private EntityManager entityManager;
	private EntityTransaction transaction;
	
	
	@BeforeEach
	void setUp() {
		entityManager = entityManagerFactory.createEntityManager();
		transaction = entityManager.getTransaction();
	}
```

테스트 코드 Spring Boot JUnit LifeCycle에 대해서 알고 계신가요?
테스트의 기본적인 라이프 사이클을 알고 계신다면 위 메서드를 통해 중복된 코드를 줄일 수 있을 것 같습니다.